### ISSUE TITLE
Compilation fails after the latest TexLive update

### ISSUE THREAD

**Issue by Vintodrimmer** [posquit0/Awesome-CV#221]
Good day,

I just ran `tlmgr update --all` and the same file I compiled the moment before started showing the following error:
```
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!
! LaTeX error: "xparse/command-already-defined"
!
! Command '\FA' already defined!
!
! See the LaTeX3 documentation for further information.
!
! For immediate help type H <return>.
!...............................................

l.166 \newfontfamily
                    \headerfont[
```

It seems to work if I comment out the `\newfontfamily\FA{FontAwesome}` line in `awesome-cv.cls`.

**Comment by ima18h** [comment]
I have this as well. I added another A so i can compile...
```
\newfontfamily\FA{FontAwesome}
```
to
```
\newfontfamily\FAA{FontAwesome}
```

**Comment by Siltyx** [comment]
Got this problem as well, the FAA trick lets me compile as well, thank you

**Comment by cbarrick** [comment]
I think this is a duplicate of #181. The error is different, but the underlying reason is the same.

**Comment by OJFord** [comment]
Closed via #344.
